cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -145,12 +148,15 @@ public void create(HttpRequest request, HttpResponder responder, @PathParam("nam
   @DELETE
   @Path("/namespaces/{namespace-id}")
   public void delete(HttpRequest request, HttpResponder responder, @PathParam("namespace-id") String namespace) {
+    if (!conf.getBoolean(Constants.Dangerous.UNRECOVERABLE_RESET,
```

Comment:
If we're having ACLs/Authorization in 2.8, we must remove this. totally agree. Just want to be sure not to release without either of those two.